### ISSUE TITLE
Clarify initial setup steps

### ISSUE THREAD

**Issue by firewave** [johnthagen/clion-cppcheck#58]
The plugin has just an average rating at https://plugins.jetbrains.com/plugin/8143-cppcheck/reviews.

It seems most of them stem from the documentation for the initial setup being confusing. We need to figure out what confuses people and improve the documentation accordingly. Maybe even rewrite it from scratch.

**Comment by johnthagen** [comment]
Agreed. For what it's worth, most of the poor ratings came before I made the landing page a lot more clear. But overall, if the plugin better guided a user through what they needed to do (such that you didn't really need to refer to docs) that would probably be best.

When I initially wrote this plugin I was new to IntelliJ plugins and wrote this one based on an existing Cpplint plugin that followed a similar usage flow. I think this workflow could definitely be improved with some more digging into what IntelliJ offers (years later).

**Comment by firewave** [comment]
I am planning on making the configuration regarding the `--language=` option a bit easier for 1.6.0 by handling it internally.

The enhanced error balloon should probably also help in some situations.

I also have a document ready which covers most of the known issues in detail as well as offering some workaround. That should hopefully resolve some user issues as well. While I was writing that I actually thought of lots of things which resulted in those past few issues I added.

I am still not sure how to integrate it with the plugin description for the CLion repository though.
